components/scream/cime_config/namelist_defaults_scream.xml

```diff
@@ -31,6 +31,8 @@ tweaking their $case/namelist_scream.xml file.
       <partmethod nlev="72">420</partmethod>
 
     Selector info will not be duplicated in the processed $case/namelist_scream.xml file.
+    If multiple parameter elements have selectors that evaulate to True, then the last
```

Comment:
I think the comment might be misleading for some one. It's not the last selector within a single element that takes precedence, but the last element among those (with the same name) that have all selectors that eval to true. I think a mini example might help. Something tiny, like
```
If bar=3 and baz=4, then
   <foo bar="3">my_foo</foo>
   <foo baz="4">your_foo</foo>
would generate an xml with content
   <foo>your_foo</foo> 
```
And maybe we should add a warning, cause this can lead to defaults being inadvertently shadowed, depending on the order in scream_namelists_defaults.xml

Reply:
Actually, we can probably remove this sentence, since you go over the rules later, and rule 1 clearly states this fact already.

Reply:
I removed the sentence; it wasn't even using the right terminology.